resources.go

```diff
@@ -1042,8 +1042,9 @@ func Provider() tfbridge.ProviderInfo {
 				DeleteBeforeReplace: true, // only 1 mount target per AZ.
 			},
 			// ECS for Kubernetes
-			"aws_eks_cluster":    {Tok: awsResource(eksMod, "Cluster")},
-			"aws_eks_node_group": {Tok: awsResource(eksMod, "NodeGroup")},
+			"aws_eks_cluster":         {Tok: awsResource(eksMod, "Cluster")},
+			"aws_eks_node_group":      {Tok: awsResource(eksMod, "NodeGroup")},
+			"aws_eks_fargate_profile": {Tok: awsResource(eksMod, "FargateProfile")},
```

Comment:
Can we mark the `fargate_profile_name` property of this resource as automaker?  For some reason they used that instead of the normal `name`.

Reply:
done! PR update on way